cassiopeia/simulator/Cas9LineageTracingDataSimulator.py

```diff
@@ -71,17 +71,29 @@ class Cas9LineageTracingDataSimulator(LineageTracingDataSimulator):
     Args:
         number_of_cassettes: Number of cassettes (i.e., arrays of target sites)
         size_of_cassette: Number of editable target sites per cassette
-        mutation_rate: Exponential parameter for the Cas9 cutting rate. The user
-            can either pass in a float, in which every site will mutate at
-            the same rate, or a list with an entry for each cut site indicating
-            potentially a different mutation rate for every site.
+        mutation_rate: Exponential parameter for the Cas9 cutting rate. Can be
+            a float, or a list of floats of length `size_of_cassette` or 
+            `number_of_cassettes * size_of_cassette`:
+                float - all sites mutate at the specified rate.
+                list of length `size_of_cassette` - each site will mutate at 
+                    the specified rate across all cassettes.
+                list of length `number_of_cassettes * size_of_cassette` - each
+                    site and cassettes will mutate at the specified rate.
         state_generating_distribution: Distribution from which to simulate state
             likelihoods. This is only used if mutation priors are not
             specified to the simulator.
         number_of_states: Number of states to simulate
-        mutation_priors: A mapping from state to probability that a user
-            can specify. If this argument is not None, states will not be
-            pulled from the state distribution.
+        mutation_priors: An optional dictionary mapping states to their prior 
```

Comment:
`mutation_priors` doesn't show up in the argument list. Is this just an instance variable, or do you want it actually to be an argument?  I think you might mean for this to be `state_priors`.

If this is the case, down in the argument list, `state_priors` typing should be updated.